### ISSUE TITLE
RHEL: salt-minion runs with user site packages excluded

### ISSUE THREAD

**Issue by chaudum** [saltstack/salt-pack-py3#207]
Even though I thought the `-s` Python interpreter option in the shebang was removed before the 2019.2.4 build, it seem it is still present in the RPM (downloaded from https://repo.saltstack.com/py3/redhat/8/x86_64/2019.2/ )

Commit that removes the `-s` shebang option`: https://github.com/saltstack/salt-pack-py3/commit/e740e9ece08e63dd53a859a729c72e891322b726#diff-45755da3bc7ba0151282e1e65c8fcbb5L229-R227

This causes problems, e.g. if you want to use salt with boto.

```console
$ rpm2cpio salt-minion-2019.2.4-1.el8.noarch.rpm | cpio -idmv
./etc/salt/minion
./etc/salt/minion.d
./etc/salt/pki/minion
./etc/salt/proxy
./usr/bin/salt-call
./usr/bin/salt-minion
./usr/bin/salt-proxy
./usr/lib/systemd/system/salt-minion.service
./usr/lib/systemd/system/salt-proxy@.service
./usr/share/man/man1/salt-call.1.gz
./usr/share/man/man1/salt-minion.1.gz
./usr/share/man/man1/salt-proxy.1.gz
147 blocks

$ cat usr/bin/salt-minion
#!/usr/bin/python3.6 -s
'''
This script is used to kick off a salt minion daemon
'''

import salt.utils.platform
from salt.scripts import salt_minion
from multiprocessing import freeze_support
...
```

**Comment by dmurphy18** [comment]
@chaudum The fix went into the 3000 release, hence I suggest upgrading, since the branch 2019.2 has only had CVE fixes applied since the 2019.2.2 point release.  And by their nature CVE fixes only contain changes for the CVE.

**Comment by chaudum** [comment]
@dmurphy18 Thanks for the clarification.
